dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/BaseAnalyticalObject.java

```diff
@@ -1424,8 +1426,14 @@ public Set<String> getSubscribers()
         return subscribers;
     }
 
+    @CheckForNull
```

Comment:
From CheckForNull javadoc:
```

/**
 * The annotated element might be null, and uses of the element should check for null.
 * <p>
 * When this annotation is applied to a method it applies to the method return value.
 */

```
so what is the effect of it on a void method ?

Reply:
It should be part of the parameter actually. Good catch.